commcare_connect/opportunity/management/commands/populate_completed_work_user_visits.py

```diff
@@ -22,6 +22,7 @@ def handle(self, *args, **options):
                     opportunity=access.opportunity,
                     user=access.user,
                     deliver_unit__in=payment_unit.deliver_units.all(),
+                    is_trial=False,
```

Comment:
Can you add this change to the `completed_work` model migration also. The migration contains the code to populate the model and was recently changed.

Reply:
Which migration are you referring to? The 0033 one? Isn't that already applied?

Reply:
Yeah, that one was already applied but I had to add a fix some days ago for that migration and I added code to migrate the completed work to the model itself. So this line of code will be required in that migration also.

https://github.com/dimagi/commcare-connect/blob/b15b4679dfdebfe1c71f85ed02d379666877ef8e/commcare_connect/opportunity/migrations/0033_completedwork_uservisit_completed_work.py#L7-L29

Reply:
Ah, I see. Cool